http/src/main/java/io/micronaut/http/MutableHttpRequestWrapper.java

```diff
@@ -38,20 +33,20 @@
  * @since 4.0.0
  */
 @Internal
-class MutableHttpRequestWrapper<B> extends HttpRequestWrapper<B> implements MutableHttpRequest<B> {
+public class MutableHttpRequestWrapper<B> extends HttpRequestWrapper<B> implements MutableHttpRequest<B> {
     private ConversionService conversionService;
 
     @Nullable
     private B body;
     @Nullable
     private URI uri;
 
-    MutableHttpRequestWrapper(ConversionService conversionService, HttpRequest<B> delegate) {
+    public MutableHttpRequestWrapper(ConversionService conversionService, HttpRequest<B> delegate) {
```

Comment:
does the constructor need to be public? The static method can be used no?

Reply:
There's some subclasses that extend it

Reply:
Made it protected